app/views/brexit_checker/_results_citizen_actions.html.erb

```diff
@@ -4,21 +4,21 @@
       <%= t('brexit_checker.results.audiences.citizen.heading') %>
     </h2>
   </header>
-  <section class="brexit-checker-audience">
-    <% citizen_results_groups.each.with_index do |group, group_index| %>
+  <section class="brexit-checker-audience brexit-checker-audience-citizen">
+    <% citizen_results_groups.each.with_index do |grouping, group_index| %>
```

Comment:
@koetsier as noted, group might not be the best name here, gone with grouping which seems marginally better?

Up for any better names? This is basically a hash that contains eveything we need to make one of these group sections including:
```
{
  group: [An instance of a group model, used to populate titles etc],
  actions: [An array of actions relevant to this group],
  criteria: [An array of relevant criteria]
}
```